remote/bitbucket/convert.go

```diff
@@ -186,5 +187,18 @@ func convertPushHook(hook *internal.PushHook, change *internal.Change) *model.Bu
 		build.Event = model.EventPush
 		build.Ref = fmt.Sprintf("refs/heads/%s", change.New.Name)
 	}
+	if len(change.New.Target.Author.Raw) != 0 {
+		build.Email = extractEmail(change.New.Target.Author.Raw)
+	}
 	return build
 }
+
+// extracts the email from a git commit author string
+func extractEmail(gitauthor string) (author string) {
+    re := regexp.MustCompile("<(.*)>")
```

Comment:
this can be moved to a global variable so that we only have to parse the regular expression once. Otherwise PR looks good, thanks!